datalad/distribution/get.py

```diff
@@ -262,6 +270,9 @@ def __call__(
                         relpath(opj(ds_path, local_results[i]['file']), ds.path)
 
             global_results.extend(local_results)
+
+        if not found_an_annex:
+            lgr.warning("Found no annex. Could not perform any get operation.")
```

Comment:
just cherry pick that commit ENHing `single_or_plural` please.

ok -- I will leave you alone for now ;)